D4Connect.cc

```diff
@@ -60,6 +60,9 @@ using namespace std;
 
 namespace libdap {
 
+const auto DAP4_CE_QUERY_KEY = "dap4.ce";
+const auto DAP4_CE_CHECKSUM_KEY = "dap4.checksum";
```

Comment:
I would need read over the code more closely, but does D4Connect not support the dap4.function part of the CE/spec. However, these changes do make D4Connect better. That code needs serious TLC...

Reply:
As far as I can see Connect knows naught about **dap4.function**